CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Typedoc for generating code documentation.
 #### Updated
 - Updated the Panel components to take in new props and using the Header from Reusable Components.
+- Updating the PlsIDForm to use an Id prop to render unique Ids on the page.
 
```

Comment:
Let's not phrase this as "render on the page," given that the ID is only visible to screen readers; maybe just, like, "accept a unique ID prop for accessibility purposes."  Also, for the sake of consistency with the rest of the changelog: capitalize "ID", and use past tense ("used the Header", "updated the PlsIDForm").